Unix/base/user.c

```diff
@@ -41,84 +40,6 @@
 
 #if defined(CONFIG_POSIX)
 static int  s_ignoreAuthCalls = 0;
-#endif
-
-#if defined(CONFIG_OS_WINDOWS)
-/* 
-    Validates user name and password;
-    Returns:
-    '0' if user account is valid and authorized to use CIM server
-    '-1' otherwise
-*/
-int AuthenticateUser(const char* user, const char* password)
-{
-    MI_UNUSED(user);
-    MI_UNUSED(password);
-    return 0;
-}
-
-/* 
-    Validates user's account for correct account name, expiration etc.
-    Returns:
-    '0' if user account is valid and authorized to use CIM server
-    '-1' otherwise
-*/
-int ValidateUser(const char* user)
-{
-    MI_UNUSED(user);
-    return 0;
-}
-
-int LookupUser(const char* user, uid_t* uid, gid_t* gid)
-{
-    MI_UNUSED(user);
-    *uid = -1;
-    *gid = -1;
-    return 0;
-}
-
-int GetUserGidByUid(uid_t uid, gid_t* gid)
-{
-    MI_UNUSED(uid);
-    *gid = -1;
-    return 0;
-}
-
-
-int GetUIDByConnection(int fd, uid_t* uid, gid_t* gid)
-{
-    MI_UNUSED(fd);
-    *uid = -1;
-    *gid = -1;
-    return -1;
-}
-
-/*
-    Creates file with random data owned by user and RO by user only
-    Parameters:
-    uid - user ID
-    size - number of bytes to write
-    path - [out] - resulting file name
-
-    Returns:
-    0 if operation was successful; -1 otherwise
-*/
-_Use_decl_annotations_
-int CreateAuthFile(
-    uid_t uid, 
-    char* content, 
-    size_t size, 
-    char path[PAL_MAX_PATH_SIZE])
-{
-    MI_UNUSED(uid);
-    MI_UNUSED(size);
-    MI_UNUSED(path);
-    MI_UNUSED(content);
-
-    return -1;
-}
-
-#elif defined(CONFIG_POSIX)
```

Comment:
based on our comment should this #elif not be a $if now?

Reply:
I believe this is ok.  No change.